src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1524,7 +1524,7 @@ public int Compare(Symbol fst, Symbol snd)
                     node = node.Parent;
                 }
 
-                CSDiagnosticInfo info = NotFound(where, simpleName, arity, where.ToString(), diagnostics, aliasOpt, qualifierOpt, options);
+                CSDiagnosticInfo info = NotFound(where, simpleName, arity, simpleName, diagnostics, aliasOpt, qualifierOpt, options);
                 return new ExtendedErrorTypeSymbol(qualifierOpt ?? Compilation.Assembly.GlobalNamespace, simpleName, arity, info);
```

Comment:
There are no compiler error codes between 1113 and 1501, so I'm not sure what you're referring to.  From ErrorCode.cs:

```cs
        ERR_ExplicitExtension = 1112,
        ERR_ValueTypeExtDelegate = 1113,
        // unused 1114-1199
        // Below five error codes are unused.
        // WRN_FeatureDeprecated2 = 1200,
        // WRN_FeatureDeprecated3 = 1201,
        // WRN_FeatureDeprecated4 = 1202,
        // WRN_FeatureDeprecated5 = 1203,
        // WRN_OldWarning_FeatureDefaultDeprecated = 1204,
        // unused 1205-1500
        ERR_BadArgCount = 1501,
        //ERR_BadArgTypes = 1502,
        ERR_BadArgType = 1503,
        ERR_NoSourceFile = 1504,
```